common/src/main/java/org/apache/uniffle/common/ShuffleIndexResult.java

```diff
@@ -74,7 +78,15 @@ public boolean isEmpty() {
 
   public void release() {
     if (this.buffer != null) {
-      this.buffer.release();
+      try {
+        this.buffer.release();
+      } catch (Throwable e) {
+        LOG.warn(
```

Comment:
Could you log the `Throwable e`, too?

Reply:
Sure

Reply:
You should catch specific exception class instead of Throwable.